py3status/modules/netdata.py

```diff
@@ -59,12 +59,25 @@ class Py3status:
     low_traffic = 400
     med_speed = 60
     med_traffic = 700
-    nic = 'eth0'
+    nic = None
 
     def __init__(self):
         self.old_transmitted = 0
         self.old_received = 0
 
+    def post_config_hook(self):
+        if self.nic is None:
+            """ Get default gateway directly from /proc."""
+            with open("/proc/net/route") as fh:
+                for line in fh:
+                    fields = line.strip().split()
+                    if fields[1] == '00000000' and int(fields[3], 16) & 2:
+                        self.nic = fields[0]
+                        break
+        if self.nic is None:
+            self.nic = 'lo'
+            self.py3.notify_user('netdata: cannot find a nic.')
```

Comment:
this block (`if ....`)should be indented one more level.

Also please add `self.py3.log('using nic %s' % self.nic)` so it will make debugging easier